DotNet.ICTICT449-VCS/Program.cs

```diff
@@ -81,6 +83,8 @@ public void ListUserOptions(List<IAscii> asciiItems)
     public void ProcessUserInput(List<IAscii> asciiItems)
     {
         var userInput = string.Empty;
+
+        // Remove 1 from user input to match what they actually meant to select.
```

Comment:
You haven't done what this comment suggests; you will need to remove 1 from the user input after the try pass.